### PR TITLE
Create date_and_time.md

### LINKED ISSUE #16: Python snippet to check current date and time
# Description
Python snippet to check current date and time using Datetime module

## Type of issue
- NOTE: *These boxes can be checked using **`[X]`***

- [X] Feature (New Script)
- [ ] Bug
- [ ] Documentation

## Checklist:

- [X] I have read the project guidelines.
- [X] I have checked all the existing projects, before submitting a new project issue.
- [X] I have checked previous issues to avoid duplicates.
- [X] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by vanigupta20024**
# Description

Created snippet to show current date and time in python and explained as well.

A short summary of what is included in your Pull Request.

Fixes #16 
## Type of change

Choose one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: snippets/python/date_and_time.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+`datetime`: The datetime module supplies classes for manipulating dates and times.
+
+### Snippet 
+```
+import datetime 
+
+# now() method returns current datetime object
+cur_time = datetime.datetime.now()  
+
+print("Date: ", cur_time.day, "/", cur_time.month, "/", cur_time.year)
```
Comment (AdityaJ7) [inline snippets/python/date_and_time.md]: Please use f strings to make it more pythonic
you can use this 
`print(f"Date: {cur_time.day} / {cur_time.month} / {cur_time.year}")`

File: snippets/python/date_and_time.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+print("Time: ", cur_time.hour, "h:", cur_time.minute, "m:", cur_time.second, "s")
```
Comment (AdityaJ7) [inline snippets/python/date_and_time.md]: Use f-strings here as well

**Comment by vanigupta20024** [comment]
Done! Thanks for letting me know this, did not know about is earlier!

**Comment by AdityaJ7** [comment]
> Done! Thanks for letting me know this, did not know about is earlier!

No problem.

Nice job with the snippet 👍🏽

**Review by AdityaJ7 — APPROVED** [review]

**Pull request merged**